### PR TITLE
Fix for #1390

### PR DESCRIPTION
**Pull request by TanyaM**
## Purpose
Resolves https://github.com/wso2/docs-apim/issues/1390

### CODE REVIEW

File: en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md [hunk 1]
```diff
@@ -474,6 +474,18 @@ are reflected in the below mentioned config files, according to the new configur
 <p>See <a href="{{base_path}}/administer/product-configurations/default-product-ports/">Default Product Ports</a> for
  a list of common and product-specific ports used by WSO2 products.</p>
 </div>
+<div class="panelHeader" style="border-bottom-width: 1px;">
+<strong>Firewall between Traffic Manager and Gateway</strong>
+</div>
+<p>If you have a firewall between the Traffic Manager and the Gateway, you need to configure the heartbeat value to keep the JMS connection alive. To configure this open the <code>&lt;PRODUCT_HOME&gt;/repository/conf/advanced/qpid-config.xml</code> file and set the heartbeat value to a non zero value as shown below.</p>
+<div class="codeContent panelContent pdl">
+```
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md:482]:
``` 
is not valid when using HTML

File: en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md [hunk 2]
```diff
@@ -474,6 +474,18 @@ are reflected in the below mentioned config files, according to the new configur
 <p>See <a href="{{base_path}}/administer/product-configurations/default-product-ports/">Default Product Ports</a> for
  a list of common and product-specific ports used by WSO2 products.</p>
 </div>
+<div class="panelHeader" style="border-bottom-width: 1px;">
+<strong>Firewall between Traffic Manager and Gateway</strong>
+</div>
+<p>If you have a firewall between the Traffic Manager and the Gateway, you need to configure the heartbeat value to keep the JMS connection alive. To configure this open the <code>&lt;PRODUCT_HOME&gt;/repository/conf/advanced/qpid-config.xml</code> file and set the heartbeat value to a non zero value as shown below.</p>
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md:480]: Shouldn't &lt;PRODUCT_HOME&gt; be &lt;API-M_HOME&gt; ?

Comment (TanyaM) [reply en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md:480]: PRODUCT_HOME is used in other places in this page and API-M_HOME is not used at all in this page. Hence used PRODUCT_HOME

File: en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md (hunk 1 above)
Comment (TanyaM) [reply en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md:482]: It is rendering HTML content properly. See the image below.
firewall

Comment (Mariangela) [reply en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md:482]: Screenshot 2020-06-24 at 17 13 23
I was looking at the rich diff and it showed as a formatting issue. Sorry for the noise.

**Comment by TanyaM** [comment]
These changes were moved to master branch ( by that time 3.2.0 was not released ) and the respective PR's can be found in [this](https://github.com/wso2/docs-apim/issues/1412) issue.

Due to the implementation level changes in 3.2.0, the content of 3.1.0 might not be accurate for 3.2.0. Hence it would be better to review the 3.2.0 considering the implementation details of 3.2.0.

**Pull request merged**